analyzers/rspec/cs/S2971.json

```diff
@@ -19,5 +18,6 @@
   "ruleSpecification": "RSPEC-2971",
   "sqKey": "S2971",
   "scope": "All",
-  "quickfix": "unknown"
+  "quickfix": "unknown",
+  "title": "LINQ expressions should be simplified"
```

Comment:
Why did this moved itself down?

Reply:
The title field was extracted in it's own metadata file since now also a Dart rule has been added for the same Id.
When metadata for csharp are merged with common metadata it adds the title at the end.